data/identify.txt

```diff
@@ -842,9 +842,10 @@ ep_only = true
 signature = 55 8B EC 6A FF 68 10 F2 40 00 68 74 9D 40 00 64 A1 00 00 00 00 50 64 89 25 00 00 00 00 83 EC 58
 ep_only = true
 
-[Armadillo v1.xx - v2.xx]
-signature = 55 8B EC 53 8B 5D 08 56 8B 75 0C 57 8B 7D 10 85 F6
-ep_only = true
+; False positive - https://www.zscaler.com/blogs/research/your-windows-8-packed
+;[Armadillo v1.xx - v2.xx]
+;signature = 55 8B EC 53 8B 5D 08 56 8B 75 0C 57 8B 7D 10 85 F6
+;ep_only = true
```

Comment:
Why not just remove this instead of commenting them out? We generally don't allow blog advertisements etc unless they are related to vulnerabilities or similar, and this approach would be similar to what you did with https://github.com/rapid7/rex-bin_tools/pull/3